client/settings/advanced-settings-section/optimized-checkout-feature.js

```diff
@@ -27,6 +29,20 @@ const SinglePaymentElementFeature = () => {
 		setLocalOCTitle( OCTitle );
 	}, [ OCTitle ] );
 
+	useEffect( () => {
+		if ( ! headingRef.current ) {
+			return;
+		}
+
+		const { highlight } = getQuery();
+		if ( highlight === 'enable-oc' ) {
```

Comment:
Is it not worth using `enable-optimized-checkout` as the query value we're looking for? It's more reader-friendly, and won't require anyone to remember what `oc` represents.
```suggestion
		if ( highlight === 'enable-optimized-checkout' ) {
```